pkg/sqlite/query.go

```diff
@@ -127,6 +127,18 @@ func (qb *queryBuilder) join(table, as, onClause string) {
 		table:    table,
 		as:       as,
 		onClause: onClause,
+		joinType: "LEFT",
```

Comment:
Same change as above, I would rename `join` to `leftJoin` for clarity.

Reply:
Same answer too. I'll work on that